src/qibo/models/dbi/utils_analytical.py

```diff
@@ -0,0 +1,215 @@
+import math
+from typing import Optional
+
+import numpy as np
+
+from qibo.models.dbi.utils import commutator, covariance, variance
+
+
+def dGamma_di_Pauli(dbi_object, n: int, Z_i: np.array, d: np.array):
```

Comment:
```suggestion
def dGamma_di_Pauli(dbi_object, n: int, Z_i: np.array, d: np.array):
```
According to PEP8 for functions it is better to use lower case letters.